### PR TITLE
Sadik mixer

### PR DESCRIPTION
**Pull request by SAkkaya**
second pull request- After Method was added

### CODE REVIEW

File: src/test/java/Test/Mixer_Test.java [hunk 1]
```diff
@@ -54,8 +55,15 @@ public void mixerTest() throws InterruptedException {
...
+
 //        9. Verify popup page’s subtotal and check out’s subtotal should be same
 
 
+
+    }
+
+    @AfterMethod
+    public void tearDown(){
+        driver.quit();
```
Comment (mrbrooks06) [inline src/test/java/Test/Mixer_Test.java:67]: you should add sleep time

File: src/test/java/Test/Mixer_Test.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+//        8. Click view cart & checkout
+
+//        9. Verify popup page’s subtotal and check out’s subtotal should be same
+
+
+
+    }
+
+    @AfterMethod
+    public void tearDown(){
```
Comment (Baros2535) [inline src/test/java/Test/Mixer_Test.java:66]: Please add 
Thread.sleep(3000); before quit

Comment (mrbrooks06) [reply src/test/java/Test/Mixer_Test.java:66]: add sleep time

File: src/test/java/Test/Mixer_Test.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+//        2. Enter your item and click search
+        driver.findElement(By.cssSelector("#twotabsearchtextbox")).sendKeys("mixer");
+        driver.findElement(By.cssSelector("#nav-search-submit-button")).click();
+
+//        3. Sort by rating and click most popular
+
+        driver.findElement(By.cssSelector(".a-dropdown-prompt")).click();
+        List<WebElement> dropDownElements = driver.findElements(By.className("a-dropdown-item"));
+        System.out.println("dropDownElements.size() = " + dropDownElements.size());
+
```
Comment (SAkkaya) [inline src/test/java/Test/Mixer_Test.java:33]: sleeep

File: src/test/java/Test/Mixer_Test.java [hunk 4]
```diff
@@ -54,8 +55,15 @@ public void mixerTest() throws InterruptedException {
         System.out.println("priceBeforeChart = " + priceBeforeChart);
```
Comment (mrbrooks06) [inline src/test/java/Test/Mixer_Test.java:56]: more space

**Comment by Baros2535** [comment]
good job

**Comment by Baros2535** [comment]
Good Job

**Pull request merged**